test/e2e/operator_test.go

```diff
@@ -225,6 +225,17 @@ func TestClusterOperatorStatusRelatedObjects(t *testing.T) {
 		},
 	}
 
+	if gatewayAPIEnabled, err := isFeatureGateEnabled(features.FeatureGateGatewayAPI); err != nil {
+		t.Fatalf("Failed to look up %q featuregate: %v", features.FeatureGateGatewayAPI, err)
```

Comment:
nit: The capitalized beginning isn't consistent with other test logging.  
```suggestion
		t.Fatalf("failed to look up %q featuregate: %v", features.FeatureGateGatewayAPI, err)
```

Reply:
The same issue came up recently here:  https://github.com/openshift/cluster-ingress-operator/pull/1152#discussion_r1994834172